src/Bridge/Doctrine/Persister/ObjectManagerPersister.php

```diff
@@ -53,7 +55,7 @@ public function __construct(ObjectManager $manager)
     public function persist($object)
     {
         if (null === $this->persistableClasses) {
```

Comment:
You can use `$objectManager = $this->managerRegistry->getManagerForClass($class);` with remove all the complexity of `persistableClass`.

It returns `ObjectManager|null`